uefi/src/proto/tcg/v1.rs

```diff
@@ -60,7 +60,7 @@ impl BootServiceCapability {
     pub fn hash_algorithm(&self) -> HashAlgorithm {
         // Safety: the value should always be 0x1 (indicating SHA-1), but
         // we don't care if it's some unexpected value.
-        unsafe { HashAlgorithm::from_bits_unchecked(u32::from(self.hash_algorithm_bitmap)) }
+        unsafe { transmute(self.hash_algorithm_bitmap as u32) }
```

Comment:
I think this can be changed to:

```rust
HashAlgorithm::from_bits_retain(u32::from(self.hash_algorithm_bitmap))
```

The safety comment can then be dropped. This used to be unsafe in bitflags v1, but they decided that was a mistake and changed it in v2: https://github.com/bitflags/bitflags/issues/292

Reply:
thanks!